test/replication_connection_test.exs

```diff
@@ -50,6 +50,28 @@ defmodule ReplicationTest do
       {:noreply, [reply], pid}
     end
 
+    @impl true
+    @doc """
+    This is part of the "stream_continuation" test and handles the COPY :done
+    state. It will start another stream right away by starting the replication
+    slot.
+    """
```

Comment:
```suggestion
    # This is part of the "stream_continuation" test and handles the COPY :done
    # state. It will start another stream right away by starting the replication
    # slot.
```